examples/qtree_example.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"go-ontap-sdk/ontap"
+	"time"
+)
+
+func main() {
+	// Initialize the ONTAP client
```

Comment:
I will write it just once for all of the comments: 
comments shouldn't be included in the merge request for several reasons.